main.go

```diff
@@ -262,16 +263,16 @@ func Main() error {
 				if err := cmd.Run(); err != nil {
 					return fmt.Errorf("%q: %w", cmd.Args, err)
 				}
-				cmd = exec.CommandContext(ctx,
-					"sed", "-i", "-e",
-					(`/timestamp "github.com\/golang\/protobuf\/ptypes\/timestamp"/ {s,timestamp.*$,timestamp "github.com/godror/knownpb/timestamppb",}; ` +
-						`/timestamppb "google.golang.org\/protobuf\/types\/known\/timestamppb"/ {s,timestamp.*$,timestamppb "github.com/godror/knownpb/timestamppb",}; `),
-					strings.TrimSuffix(pbFn, ".proto")+".pb.go",
-				)
-				cmd.Stdout, cmd.Stderr = os.Stdout, os.Stderr
-				if err := cmd.Run(); err != nil {
-					return fmt.Errorf("%q: %w", cmd.Args, err)
-				}
+				//cmd = exec.CommandContext(ctx,
+				//	"sed", "-i", "-e",
+				//	(`/timestamp "github.com\/golang\/protobuf\/ptypes\/timestamp"/ {s,timestamp.*$,timestamp "github.com/godror/knownpb/timestamppb",}; ` +
+				//		`/timestamppb "google.golang.org\/protobuf\/types\/known\/timestamppb"/ {s,timestamp.*$,timestamppb "github.com/godror/knownpb/timestamppb",}; `),
+				//	strings.TrimSuffix(pbFn, ".proto")+".pb.go",
+				//)
+				//cmd.Stdout, cmd.Stderr = os.Stdout, os.Stderr
+				//if err := cmd.Run(); err != nil {
+				//	return fmt.Errorf("%q: %w", cmd.Args, err)
+				//}
```

Comment:
Why?

Reply:
Failing not sure why 

pkg/pb/pb.proto:5:1: warning: Import google/protobuf/timestamp.proto is unused.
sed: 1: "/timestamp "github.com\ ...": bad flag in substitute command: '}'
15:28:22.810 ERR "ERROR" attrs={"error":"[\"sed\" \"-i\" \"-e\" \"/timestamp \\\"github.com\\\\/golang\\\\/protobuf\\\\/ptypes\\\\/timestamp\\\"/ {s,timestamp.*$,timestamp \\\"github.com/godror/knownpb/timestamppb\\\",}; /timestamppb \\\"google.golang.org\\\\/protobuf\\\\/types\\\\/known\\\\/timestamppb\\\"/ {s,timestamp.*$,timestamppb \\\"github.com/godror/knownpb/timestamppb\\\",}; \" \"/Users/charbelsemaan/go/src/pkg/pb/pb.pb.go\"]: exit status 1"}

Reply:
It's better to fix those sed commands - remove the {} symbols! (or add a ; before the closing }, that's the real error - but for one command, {} grouping is not needed).